src/cli/commands/dev.ts

```diff
@@ -48,7 +48,7 @@ export class Dev implements Command {
     clearConsole()
     mylogger.info('boot')
 
-    createWatcher({
+    await createWatcher({
```

Comment:
Noticed this would-have-been-a-new-bug just now. Not tested unfortunately but dev command testing will eventually come...